core/docker/compose.sh

```diff
@@ -1,11 +1,23 @@
 composeFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
-if [ "$1" == "up" ]; then
+if [ "$1" == "init" ]; then
+    docker-compose -f "$composeFilePath"/docker-compose-mongo.yml up -d
+
+    # Set up the replica set
+    "$composeFilePath"/initiateReplicaSet.sh
+
     docker-compose -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml up -d
+elif [ "$1" == "up" ]; then
+    docker-compose -f "$composeFilePath"/docker-compose-mongo.yml up -d
+
+    # Wait for mongo replica set to be set up
+    sleep 20
+
+    docker-compose -f "$composeFilePath"/docker-compose.yml up -d
 elif [ "$1" == "down" ]; then
-    docker-compose -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml stop
+    docker-compose -f "$composeFilePath"/docker-compose-mongo.yml -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml stop
 elif [ "$1" == "destroy" ]; then
-    docker-compose -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml down -v
+    docker-compose -f "$composeFilePath"/docker-compose-mongo.yml -f "$composeFilePath"/docker-compose.yml -f "$composeFilePath"/docker-compose.dev.yml -f "$composeFilePath"/importer/docker-compose.config.yml down -v
 else
     echo "Valid options are: up, down, or destroy"
```

Comment:
@bradsawadye This `Valid options` set needs to be updated for all the relevant bash scripts

Reply:
@bradsawadye Have you addressed this?

Reply:
Hey man, so on this, the mongo script is not on that location. Its in the core component. The destroy command in the core compose script handles this

Reply:
@bradsawadye 
```suggestion
    echo "Valid options are: init, up, down, or destroy"
```